CHANGELOG.md

```diff
@@ -1,6 +1,97 @@
 # libddwaf release
 
-### v1.18.0 ([unstable](https://github.com/DataDog/libddwaf/blob/master/README.md#versioning-semantics))
+## v1.19.0 ([unstable](https://github.com/DataDog/libddwaf/blob/master/README.md#versioning-semantics))
+
+### New features
+This new version of `libddwaf` introduces a multitude of new features in order to support new use cases and expand existing ones.
+
+#### Exploit prevention: Shell injection detection
+A new operator `shi_detector` has been introduced for detecting and blocking shell injections, based on input parameters and the final shell code being evaluated. This new operator is part of the exploit prevention feature, so it is meant to be used in combination with targeted instrumentation. 
+
+The following example rule takes advantage of the new operator to identify injections originating from request parameters:
+
+```yaml
+  - id: rsp-930-004
+    name: SHi Exploit detection
```

Comment:
`SHI` instead

Reply:
The lowercase `i` means injection, like in `SQLi`.